spec/struct.dd

```diff
@@ -211,7 +211,7 @@ $(H2 $(LNAME2 POD, Plain Old Data))
     $(P A struct or union is $(I Plain Old Data) (POD) if it meets the following criteria:)
 
     $(OL
-    $(LI it is not nested)
+    $(LI it is not nested or static)
```

Comment:
This sounds weird. Maybe a more correct phrasing would be: "it is not nested or if it is nested it is declared as `static`".

Reply:
Indeed, the current wording could be interpreted as either "not (nested or static)" or "(not nested) or static"

Reply:
ok, updated.